bids-validator/validators/bids/fullTest.js

```diff
@@ -81,7 +81,7 @@ const fullTest = (fileList, options, annexed, dir, callback) => {
   })
 
   // check if dataset contains T1w
-  if (summary.modalities.indexOf('T1w') < 0) {
+  if (summary.dataTypes.includes('T1w')) {
```

Comment:
I think this is inverted...

```suggestion
  if (!summary.dataTypes.includes('T1w')) {
```